### PR TITLE
Added template details to library

### PR DESCRIPTION
**Pull request by eguzman-wisc**
Should do exactly as it's set out to, definitely increased processing time by a bit.

### CODE REVIEW

**Review by dgfitch — CHANGES_REQUESTED** [review]
OK, cool. But I want to make it a reused function instead of copy and pasting. Because I'm a stickler like that - and it will make it easier to change if we need to...

**Comment by eguzman-wisc** [comment]
Made the changes you requested!

File: selfreportlibrary.py [hunk 1]
```diff
@@ -37,25 +74,37 @@ def run(mother):
...
+    
     # Replace the "Sorted by Construct" section with our new text
     old_section = cat.get_sections(matches = "Sorted by Construct")[0]
     cat.replace(old_section, newtext)
     newpage = str(cat)
 
     if oldtext != newpage:
         logging.warning("Updating self-report category page, change detected")
-        category.save(newpage, "Automated edit to build construct categories on self-report library")
+        #category.save(newpage, "Automated edit to build construct categories on self-report library")
```
Comment (dgfitch) [inline selfreportlibrary.py]: You've got this commented out now, but the details function looks great!

**Comment by dgfitch** [comment]
Let me know if you see issues with this merge, I'll check it out tomorrow.

**Pull request merged**